docs/_docs/Chrome/attr.md

```diff
@@ -0,0 +1,33 @@
+---
+title: .attr
+category: Chrome
+---
+
+The `attr` method operates similarly to the jQuery `attr` method, and return the value of an attribute of a DOM element. It's called with 2 parameters: the css-style selector of the element you wish to query, and the attribute you want to retrieve.
+
+*JavaScript*
+```js
+const { Chrome } = require('navalia');
+const chrome = new Chrome();
+
+chrome.goto('https://joelgriffith.github.io/navalia/')
```

Comment:
I think it's missing a corresponding comment above this block (`// Getting an attribute`).